input/pagecontent/reference.md

```diff
@@ -924,7 +918,7 @@ The following table is a summary of the rules that may apply to profiles, extens
 | Contains (for standalone extensions) [3]| <code>* &lt;Extension&gt; contains {Extension name|id|url} named {name} {card} <i>{flags}</i></code> <br/>  <code> * &lt;Extension&gt; contains {Extension1 name|id|url} named {name1} {card1} <i>{flags1}</i> and {Extension2 name|id|url} named {name2} {card2} <i>{flags2}</i> ...</code>
 | Contains (for slicing) [3]| <code>* &lt;array&gt; contains {name} {card} <i>{flags}</i></code> <br/> <code>* &lt;array&gt; contains {name1} {card1} <i>{flags1}</i> and {name2} {card2} <i>{flags2}</i> ...</code>|
 | Flag | `* <element> {flag}` <br/> `* <element> {flag1} {flag2} ...` <br/> `* <element1> and <element2> and <element3> ... {flag1} {flag2} ...` |
-| Insert | <code>* insert {RuleSet name}<i>({value1}, {value2}, ...)</i></code> |
+| Insert | <code>* insert {RuleSet name}<i>({value1}, {value2}, ...)</i></code> <br/>  {%include tu.html%} <code>* &lt;element&gt; insert {RuleSet name}<i>({value1}, {value2}, ...)</i></code>|
```

Comment:
I think this would look better w/ the `TU` _after_ the syntax.

For comparison... `TU` before:
<img width="592" alt="image" src="https://user-images.githubusercontent.com/2278253/125122410-61dc3c80-e0c3-11eb-9264-d8d74490de2a.png">
`TU` after:
<img width="553" alt="image" src="https://user-images.githubusercontent.com/2278253/125122366-512bc680-e0c3-11eb-9d80-c13a5b0e9c3e.png">

Reply:
Actually it is clearer if the row is split into two rows. And I moved the TUs uniformly to the left margins, where they visually read better.